### DISCUSSION TITLE
Monet shuttered, now what?

### DISCUSSION THREAD

**Discussion by J-HS-L** [termux/termux-app#4271] [Q&A] [answered]
1.
Now that Monet's been dropped, are users bestoff migrating to mainline termux soon-ish?

2.
Does mainline have plans to implement every/any-thing it lacks, compared to Monet?

Thank you!

**Comment by Biswa96** [comment]
> Does mainline have plans to implement every/any-thing it lacks, compared to Monet?

What are those features?

**Reply by luisdavim** [reply]
I think this is one of them for example:
https://github.com/termux/termux-app/pull/2973

BTW, there seems to be lots of good stuff in the PRs would be nice to get some of them merged and added new release created.

**Reply by Biswa96** [reply]
You can build the termux-app project with the patch file.

**Reply by luisdavim** [reply, marked as answer]
Sure, or I could create my own fork, but that's not the point, the point is to make these features available to all termux users.

**Reply by luisdavim** [reply]
Maybe now that @fornwall is back we could see more PRs getting reviewed and even merged? That should also help attract more contributors...

**Reply by J-HS-L** [reply]
> Maybe now that @fornwall is back, we could see more PRs getting reviewed and even merged? That should also help attract more contributors...

Yes please @fornwall ! 🙏

**Comment by zavocc** [comment]
it's strongly recommend to use upstream termux since the project is unmaintained although technically you can still use it as the packages are updated,  termux app gets updated too, so for the long run its best to avoid unmaintained forks, and not it's that hard to move data
